docs/Setup.md

```diff
@@ -13,16 +13,22 @@ Start by making sure the command line with Gradle works first, because **all the
 1. Install Java
     1. Install JVM 11 or greater on your system.  Our setup has been tested with Java 11-17.  Although a variety of JVM distributions are available and should work, we have settled on recommending [Adoptium/Temurin](https://adoptium.net), because this is the default distribution used by Gradle toolchains.  For Windows or Linux, be sure that the `JAVA_HOME` environment variable points to the right Java version.  Note: If you switch from a newer to an older JVM version, you may see an error like the following `> com.android.ide.common.signing.KeytoolException: Failed to read key AndroidDebugKey from store "~/.android/debug.keystore": Integrity check failed: java.security.NoSuchAlgorithmException: Algorithm HmacPBESHA256 not available`.  A solution is to delete the debug keystore and allow it to be re-generated.
     1. Android Studio has an embedded JVM, although running Gradle tasks from the command line requires a separate JVM to be installed.  Our Gradle scripts are configured to use toolchains to automatically install the correct JVM version.
-1. Configure Rust
+2. Configure Rust
     1. [Install rust](https://www.rust-lang.org/learn/get-started)
         1. macOS with Homebrew
             1. `brew install rustup`
             1. `rustup-init`
     1. Add Android targets
         ```bash
         rustup target add armv7-linux-androideabi aarch64-linux-android i686-linux-android x86_64-linux-android
-        ```
-1. Install Android Studio and the Android SDK
+3. Install python 2.7 
+   1. macOs with Homebrew
```

Comment:
I have a concern this may negatively impact the developer's environment, especially if python is being used for something else on the system already.  As an enhancement, I suggest using the environment variable used by the Rust gradle plugin to inject this in the most minimal way possible which is described here:  https://github.com/mozilla/rust-android-gradle#specifying-paths-to-sub-commands-python-cargo-and-rustc

It seems like these are two good options:

 - rust.pythonCommand in ${rootDir}/local.properties
 - the environment variable RUST_ANDROID_GRADLE_PYTHON_COMMAND